org.mixedrealitytoolkit.core/CHANGELOG.md

```diff
@@ -2,11 +2,12 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
-## [3.3.0-development] - 2024-06-24
+## [3.2.3-development] - 2024-06-24
```

Comment:
Any clue why/how this was changed to 3.3.0 in changelong, but kept asd 3.2.2 in package.json?

Reply:
https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/commit/9f1831d385880a403be4693c65331664afe5ca38